README.md

```diff
@@ -199,7 +199,8 @@ tl;dr:
 
 #### Passing command-line arguments
 
-EmuHawk takes some command-line options which aren't well-documented; you might be able to figure them out from [the source](https://github.com/TASEmulators/BizHawk/blob/78daf4913d4c8e47d24fc14d84ca33ddef913ed4/src/BizHawk.Client.Common/ArgParser.cs).
+EmuHawk takes some command-line options which, starting from 2.9.2, can be listed with `--help`.
```

Comment:
This doesn't actually work on windows release builds due to the `WinExe` target, so this will need some consideration.

Reply:
I've fixed this by caveating it with "on Linux".